### PR TITLE
Completed Persistence Module

### PR DESCRIPTION
**Pull request by ecstasy-repos**

### CODE REVIEW

File: Dashboard/Server/Persistence/TelemetryPersistence.cs [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+using System.Linq;
+using System.Xml.Serialization;
+
+
+namespace Dashboard.Server.Persistence
+{
+
+    public class TelemetryPersistence : ITelemetryPersistence
+    {
+        public ServerDataToSave RetrieveAllSeverData()
```
Comment (bruise-88) [inline Dashboard/Server/Persistence/TelemetryPersistence.cs:17]: Please add summary comment

File: Dashboard/Server/Persistence/TelemetryPersistence.cs [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+           
+            if (!Directory.Exists(p1)) Directory.CreateDirectory(p1);
+            plt1.SaveFig(Path.Combine(p1, "ChatCountVsUserID.png"));
+            response.IsSaved = true;
+            Trace.WriteLine("ChatCountVsUserID.png saved Successfully!!");
+            return response;
+            
+        }
+
+        public ResponseEntity UserCountVsTimeStamp_PlotUtil(Dictionary<int, int> UserCountAtAnyTime, string sessionId)
```
Comment (bruise-88) [inline Dashboard/Server/Persistence/TelemetryPersistence.cs:119]: Please make these functions private for abstraction

File: Testing/Dashboard/Persistence.UnitTests/PersistenceNUnitTests.cs [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+
+            Assert.IsTrue(File.Exists(Path.Combine(path, response.FileName)));
+
+            string TextActuallySaved = File.ReadAllText(Path.Combine(path, response.FileName));
+            Trace.WriteLine(TextActuallySaved);
+
+        }
+
+        [Test]
+        public void retriveAllSeverData_isReturningCorrectly()
```
Comment (bruise-88) [inline Testing/Dashboard/Persistence.UnitTests/PersistenceNUnitTests.cs:88]: function name capital

**Comment by ecstasy-repos** [comment]
I have made all required changes, Review and suggest changes if required..

**Pull request merged**